request/iframe.js

```diff
@@ -11,10 +11,11 @@ define([
 	'../dom',
 	'../dom-construct',
 	'../_base/window',
-	'../NodeList-dom'/*=====,
+	'../NodeList-dom',
+        '../NodeList-manipulate'/*=====,
 	'../request',
 	'../_base/declare' =====*/
-], function(module, require, watch, util, handlers, lang, ioQuery, query, has, dom, domConstruct, win/*=====, NodeList, request, declare =====*/){
+], function(module, require, watch, util, handlers, lang, ioQuery, query, has, dom, domConstruct, win/*=====, NodeList, NodeList, request, declare =====*/){
```

Comment:
I'm not quite sure this works as intended, as you now have two modules both mapped to NodeList... but they're both commented out...

Reply:
Yes, from the NodeList-manipulate documentation, I conclude these modules (NodeList-dom and NodeList-manipulate) add functionality to the base NodeList class, adding the `<node>.val()` function as used below.

What leads me to believe this is the section

> Doing a dojo.require(“dojo.NodeList-manipulate”) (since Dojo 1.7, it’s suggested to use AMD-style module loading, e.g. require([“dojo/NodeList-manipulate”]).) will add some addition methods to dojo.NodeList (the return object from a dojo.query call) that allow easier manipulation of HTML as it relates to the nodes in the dojo.NodeList.

In https://dojotoolkit.org/reference-guide/1.10/dojo/NodeList-manipulate.html.

Reply:
ok, so essentially what is going on here is that you don't need a local reference to these dependencies, so the return value doens't matter. I guess what I would suggest as a final change is:

1. remove lines 16-17 (the commented out request and declare dependencies, since they are not used
2. remove `/*=====, NodeList, NodeList, request, declare =====*/ from line 18

Reply:
Done and put a small explaining comment in the code about the reference not being required in the code itself.